### PR TITLE
add conda env in sourc install instructions

### PR DESCRIPTION
**Pull request by quasiben**
cc @beckernick

### CODE REVIEW

**Comment by beckernick** [comment]
👍

File: docs/source/install.rst [hunk 1]
```diff
@@ -23,11 +23,22 @@ Source
 
 The following instructions assume you'll be using ucx-py on a CUDA enabled system and is in a `Conda environment <https://docs.conda.io/projects/conda/en/latest/>`_.
 
-Note: UCX depends on the following system libraries being present: ``libibverbs``, ``librdmacm``, and ``libnuma`` (numactl on Enterprise Linux).  Please install these with your Linux system's package manager.
+.. note::
+    UCX depends on the following system libraries being present: ``libibverbs``, ``librdmacm``, and ``libnuma`` (numactl on Enterprise Linux).  Please install these with your Linux system's package manager.  Additionally, we provide an example conda environment which installs necessary dependencies for
```
Comment (pentschev) [inline docs/source/install.rst]: Maybe we should add `libibcm` for those who need IB as well?

**Comment by pentschev** [comment]
Added one small comment on `libibcm`, otherwise LGTM.

File: docs/source/install.rst (hunk 1 above)
Comment (quasiben) [reply docs/source/install.rst]: agreed

**Comment by quasiben** [comment]
rerun tests

**Comment by jakirkham** [comment]
@quasiben, do you mind if I take over this PR?

**Comment by quasiben** [comment]
please do

**Comment by jakirkham** [comment]
Thanks! 😄

File: docs/source/install.rst [hunk 2]
```diff
@@ -23,11 +23,23 @@ Source
 
 The following instructions assume you'll be using ucx-py on a CUDA enabled system and is in a `Conda environment <https://docs.conda.io/projects/conda/en/latest/>`_.
 
-Note: UCX depends on the following system libraries being present: ``libibverbs``, ``librdmacm``, and ``libnuma`` (numactl on Enterprise Linux).  Please install these with your Linux system's package manager.
+.. note::
+    UCX depends on the following system libraries being present: ``libibverbs``, ``librdmacm``, ``librdmacm``,
```
Comment (jakirkham) [inline docs/source/install.rst]: Oops missed this. Apparently we are referencing `librdmacm` twice. I'll update in PR ( https://github.com/rapidsai/ucx-py/pull/351 ).

**Pull request merged**